string-manipulation-2/src/capitalize-words.js

```diff
@@ -0,0 +1,14 @@
+/* exported capitalizeWords */
+
+function capitalizeWords(string) {
+  var yesCapitalize = '';
+
+  for (var i = 1; i < string.length; i++) {
+    if (string[i - 1] === ' ') {
+      yesCapitalize += string[i].toUpperCase();
+    } else if (string[i - 1] !== ' ') {
+      yesCapitalize += string[i].toLowerCase();
+    } else { yesCapitalize += string[i]; }
```

Comment:
This `else` will never happen. Your second condition is the opposite of your first condition, so one of those will always be met. You should just make the `else if` above into an `else`